gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,44 @@
+# Italian Mars Society's ERAS Project #
+
+## Sub-Org Details ##
+
+Sub-org admin: Antonio Del Mastro - <gsoc@marssociety.it>
+
+Main repository: <https://bitbucket.org/italianmarssociety/eras/src>
+
+Mailing list: <https://groups.google.com/forum/#!forum/erasproject>
+
+
+## Ideas ##
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+DIFFICULTY LEVEL: INTERMEDIATE / HARD
+
+MENTORS: [@aldebran](https://bitbucket.org/aldebran/) , [@OPersian](https://bitbucket.org/OPersian/)
+
+DESCRIPTION:
+
+Solar storms are responsible for disruption of satellite communication, and damage to space electronical equipments. The storms have to be taken into account also for EVA and habitat maintenance activities, as the higher levels of radiation brought by them have a detrimental effect on the crew member's health.
+
+Prediction of these storms are essential to prevent said damage. A lot of astronomical data is generated on a daily basis, and this could be used in conjunction with machine learning methods to predict solar storms.
+
+In this project, the student will be required to:
+
+* Predict solar storms to prevent damage.
+* Provide real time information about it on a web page.
```

Comment:
You should detail with technologies are expected the students to use. Is that using python connecting with other libraries like `sunpy` or it's all developed in `C`? Also, require the student to "predict solar storms" is a bit too much (in my opinion). :)
Same thing for the webpage, is it suppose to be using some particular cms or a static site like jekyll. What does the student needs to know? does the content needs to be dynamic and interactive or just static?